packages/parser/lib/css_parser.mly

```diff
@@ -487,33 +595,6 @@ complex_selector:
     }
   }
 
-/* value_in_prelude we transform WS_* into Delim with white spaces inside
-in value we transform to regular Delim
-The rest of value_in_prelude and value should be sync */
-value_in_prelude:
-  | b = paren_block(prelude) { Paren_block b }
-  | b = bracket_block(prelude) { Bracket_block b }
-  | n = percentage { Percentage n }
-  | i = IDENT { Ident i }
-  | i = TAG { Ident i }
-  | s = STRING { String s }
-  | c = COMBINATOR { Combinator c}
-  | o = OPERATOR { Operator o }
-  | d = DELIM { Delim d }
-  | DOT { Delim "." }
-  | COLON { Delim ":" }
-  | DOUBLE_COLON { Delim "::" }
-  | h = HASH { Hash h }
-  | COMMA { Delim "," }
-  | n = NUMBER { Number n }
-  | r = UNICODE_RANGE { Unicode_range r }
-  | d = FLOAT_DIMENSION { Float_dimension d }
-  | d = DIMENSION { Dimension d }
-  | v = INTERPOLATION { Variable v } /* $(Lola.value) */
-  | f = loc(FUNCTION) xs = loc(prelude) RIGHT_PAREN; { Function (f, xs) } /* calc() */
-  | u = URL { Uri u } /* url() */
-  | WS { Delim " " }
-
```

Comment:
Does deleting this causes a any problems in terms of conflicts or whitespace stuff?

If no, it's a big improvement

Reply:
> Does deleting this causes a any problems in terms of conflicts or whitespace stuff?

No, it does not!